src/edu/csus/ecs/pc2/core/Utilities.java

```diff
@@ -1978,5 +1978,63 @@ public static void setShowStandingsPanes(boolean showStandingsPanes) {
     public static boolean isShowStandingsPanes() {
         return showStandingsPanes;
     }
+    
+    /**
+     * Concatenate Arrays.
+     * 
+     * This is null-safe, arrays can be null and will return 
+     * an array.
+     * 
+     * @param one first array
+     * @param two secodn array
+     * @return a new array which contains contents of one and two arrays
+     */
+    public static String[] copyArray(String[] one, String[] two) {
```

Comment:
The name of this method is VERY misleading.  "Copy" implies that it is going to do (something like) copy array `one` into array `two` (or maybe vice-versa).  The comment at the very top of the method is the only clue to what it really does:  _concatentate_ two arrays and return the concatenated result.  But of course, clients are not (likely) going to see the _comments_; they are most likely going to look at the method _name_.

A better name would be `concatArrays()`, or perhaps `mergeArrays()`.  But I seriously think "copyArray" is a bad name and should be changed to reflect what the method actually does.

Reply:
Will update the name, note that the Java copyArray is used which is where the name was based on.